packages/runtime/src/account.ts

```diff
@@ -24,8 +24,71 @@ const globalState = "global-state";
 const localStateSchema = "local-state-schema";
 const globalStateSchema = "global-state-schema";
 
+class RuntimeAccount implements RuntimeAccountI {
+  sk: Uint8Array
+  addr: string
+  name?: string;
+  authAccount?: RuntimeAccountI;
+
+  constructor (builder: RuntimeAccountBuilder) {
+    this.sk = builder.sk;
+    this.addr = builder.addr;
+    this.name = builder.name;
+  }
+
+  rekeyTo (authAccount: RuntimeAccountI): void {
+    this.authAccount = authAccount;
+  }
+}
+
+export class RuntimeAccountBuilder {
```

Comment:
I'm not sure why do we need a builder for account? The constructor and logic for the `RuntimeAccount`. The account can be only rekeyed. I don't see how adding a builder here will help. If it's only for setting up key generation, then maybe it's overengineering?

Reply:
I think you right. We just need move `from` and `genKey` method to RunttimeAccount, code more clear and sense.